docs/bibliography.rst

```diff
@@ -0,0 +1,6 @@
+############
+Bibliography
+############
+
+.. bibliography::
+   :all:
```

Comment:
The `:all:` here makes it so that all of the references that are put into the `.bib` file are included in the bibliography, even if they're not referenced elsewhere.